### PR TITLE
[ci]: add vstest

### PR DESCRIPTION
**Pull request by lguohan**
Signed-off-by: Guohan Lu <lguohan@gmail.com>

#### What I did
add vstest

#### How to verify it
pr

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -6,74 +6,103 @@
...
-    source: specific
-    project: build
-    pipeline: 1
-    artifact: sonic-buildimage.vs
-    runVersion: 'latestFromBranch'
-    runBranch: 'refs/heads/master'
-  displayName: "Download artifacts from latest sonic-buildimage build"
+  jobs:
+  - job:
+    displayName: "Build"
```
Comment (jleveque) [inline azure-pipelines.yml:21]: Now that this job is nested in a stage called "Build", this job is displayed as "Build Build", which doesn't make much sense. I can think of a few possible solutions to make this better:

1. Change this job's `displayName` to `Python 3` or `Python 3 Wheel` or `Python Wheel`
2. Split this into two jobs, the unit test and the wheel build and define the `displayName` fields appropriately
3. Split this into two stages, `UnitTest` and `Build` and define the `displayName` fields appropriately for the jobs

Thoughts?

**Review by jleveque — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (lguohan) [reply azure-pipelines.yml]: choice 1 is good.

**Comment by lguohan** [comment]
@jleveque , can you review this pr again?

**Review by jleveque — DISMISSED** [review]
Looks good to me, but tests are failing. Is this expected?

**Comment by lguohan** [comment]
vstest passed. @jleveque , can you approve?

**Comment by lguohan** [comment]
retest this please

**Review by jleveque — APPROVED** [review]

**Pull request merged**